app/uk/gov/hmrc/tariffclassificationfrontend/models/ReportResult.scala

```diff
@@ -20,4 +20,7 @@ case class ReportResult
 (
   group: Option[String],
   value: Seq[Int]
-)
+) {
+  def size: Int = value.size
+  def average: Double = value.sum.toDouble / size
```

Comment:
Do we need to guard against size being zero?

Reply:
Count will never be zero :) there wouldnt be a ReportResult for it if it was